pkg/processor/trigger/cron/cron_test.go

```diff
@@ -166,6 +167,19 @@ func (suite *TestSuite) TestGetNextEventSubmitDelayScheduleRunsImmediatelyOnMiss
 	suite.Assert().EqualValues(0, nextEventDelay)
 }
 
+func (suite *TestSuite) TestNextScheduleDayDifference() {
```

Comment:
Add another test please, that uses `func newTrigger(...)` and provide it with 
`fmt.Sprintf("* %d %d * *", lastRuntime.Minute(), lastRuntime.Hour())`